src/events/acceleration.js

```diff
@@ -355,6 +355,7 @@ var shake_threshold = 30;
  * @param {number} value The threshold value
  */
 p5.prototype.setMoveThreshold = function(val) {
+  p5._validateParameters('setMoveThreshold', arguments);
```

Comment:
this seems like a double check... the if statement below checks if the param is a number

Reply:
this is true, however, without the `_validateParameters` call, there's no warning given to the user if the type is wrong, the argument is missing (or null), or if extra parameters are passed.

Reply:
ok, then let's remove the if check below..